thirdparty/faiss/faiss/invlists/OnDiskInvertedLists.cpp

```diff
@@ -523,7 +527,7 @@ void OnDiskInvertedLists::free_slot(size_t offset, size_t capacity) {
         it++;
     }
 
-    size_t inf = 1UL << 60;
```

Comment:
need not change this, both 1UL and 1ULL are 8 bytes.

Reply:
In windows, UL means `unsigned long` which is 4bytes(32bits) even in win64. 
Ref: https://stackoverflow.com/questions/64662711/c-64bit-compilation-on-vs-code-with-windows-10